vimeo-networking/src/main/java/com/vimeo/networking/utils/VimeoNetworkUtil.java

```diff
@@ -85,8 +84,6 @@ public static Gson getGson() {
     public static GsonBuilder getGsonBuilder() {
         // Example date: "2015-05-21T14:24:03+00:00"
         return new GsonBuilder().registerTypeAdapterFactory(new Stag.Factory())
-                .registerTypeAdapterFactory(new PrivacyDeserializer.TypeAdapterFactory())
-                .registerTypeAdapterFactory(new VideoFileDeserializer.TypeAdapterFactory())
```

Comment:
I didn't touch that method, and I didn't create it to begin with, so I'm not sure. @kvenn why is it done this way?